src/components/checkbox/Checkbox.css

```diff
@@ -0,0 +1,8 @@
+.Checkbox {

```

Comment:
Checkbox is a separate issue. Whereever possible, keep PRs focused on 1 thing, even if it's tiny. Many small PRs are easier to merge, and it avoids issue where good work can't be merged when there are issues with the other parts.

Reply:
Yeah, I know it's a separate issue, although I needed it for the files. I can separate it though and create a PR just for it and then rebase this branch.

Reply:
No need to split this one, just a note for future ones.